recorder.go

```diff
@@ -117,6 +118,9 @@ type Options struct {
 	// variable-length value types (strings, interface{}, etc).
 	MaxLogValueLen int `yaml:"max_log_value_len"`
 
+	// MaxLogsPerSpan limits the number of logs in a single span.
```

Comment:
remind me -- if someone sets this to `0`, it would actually inherit the basictracer default, correct?